audio/out/ao_pulse.c

```diff
@@ -294,6 +294,12 @@ static int pa_init_boilerplate(struct ao *ao)
     struct priv *priv = ao->priv;
     char *host = priv->cfg_host && priv->cfg_host[0] ? priv->cfg_host : NULL;
     bool locked = false;
+    pa_proplist *props = pa_proplist_new();
+
+    pa_proplist_sets(props, PA_PROP_MEDIA_ROLE, ao->init_flags & AO_INIT_MEDIA_ROLE_MUSIC ?  "music" : "video");
```

Comment:
At least according to [docs](https://www.freedesktop.org/software/pulseaudio/doxygen/proplist_8h.html#a768c59b205d817aa47c8b261bab1bfb1), these should return nonzero on failure (most likely due to pa not being able to allocate its own copy of the string?).

Whether we care is a separate question, o' course.

Reply:
2 reminds me of us previously crashing something by passing unexpected non-UTF-8 stuff into another library :)

But alright, the first case indeed cannot happen.